docs/pages/components/combobox-input.md

```diff
@@ -45,7 +45,7 @@ summary:
               <div class="fd-input-group fd-input-group--after fd-input-group--compact">
                   <input type="text" class="fd-input fd-input--compact" id="" placeholder="Select Fruit">
                   <span class="fd-input-group__addon fd-input-group__addon--after fd-input-group__addon--button">
-                      <button class=" fd-button--light sap-icon--navigation-down-arrow"></button>
+                      <button class=" fd-button--light fd-button--compact sap-icon--navigation-down-arrow"></button>
```

Comment:
Although probably a necessary change, it completely unrelated to the DatePicker button.  This should be reverted and put into its own PR.